backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/acct/api/impl/AcctSchemaDAO.java

```diff
@@ -99,12 +96,7 @@ public final AcctSchema getByCliendAndOrg(final ClientId clientId, final OrgId o
 	@Override
 	public AcctSchemaId getAcctSchemaIdByClientAndOrg(final ClientId clientId, final OrgId orgId)
 	{
-		final AcctSchemaId acctSchemaId = AcctSchemaId.ofRepoId(DB.getSQLValueEx(ITrx.TRXNAME_None, "SELECT getC_AcctSchema_ID(?,?)", clientId, orgId));
-		if (acctSchemaId == null)
-		{
-			throw new AdempiereException("No accounting schema found for " + clientId + " and " + orgId);
-		}
-		return acctSchemaId;
+		return AcctSchemaId.ofRepoId(DB.getSQLValueEx(ITrx.TRXNAME_None, "SELECT getC_AcctSchema_ID(?,?)", clientId, orgId));
```

Comment:
i'm aware that the former code didn't work, but still it makes sense to have it.
maybe it will work like this?
```suggestion
		final AcctSchemaId acctSchemaId = AcctSchemaId.ofRepoIdOrNull(DB.getSQLValueEx(ITrx.TRXNAME_None, "SELECT getC_AcctSchema_ID(?,?)", clientId, orgId));
		if (acctSchemaId == null)
		{
			throw new AdempiereException("No accounting schema found for " + clientId + " and " + orgId);
		}
		return acctSchemaId;
```

Reply:
makes sense.